proto/process/agent.proto

```diff
@@ -518,9 +518,9 @@ enum EphemeralPortState {
 }
 
 message Connection {
-	reserved 2, 3, 4, 7, 8, 9, 13, 14, 15, 35;
+	reserved 2, 3, 4, 7, 8, 9, 13, 14, 15, 34, 35;
```

Comment:
As we discussed over slack, we probably don't want to deprecate the old field until we know for sure that our backend can handle the new one.